indi-duino/add-on/weatherradio/bin/wr_config.py

```diff
@@ -24,13 +24,13 @@
 INDIDEVICEPORT="/dev/ttyUSB0"
 
 # RRD storage holding time series data
-RRDFILE="./weather.rrd"
+RRDFILE="/usr/local/share/weatherradio/weather.rrd"
```

Comment:
is there a reason this is /usr/local/share and not /usr/share? The reason is if we make a debian package for this, it should be install to /usr/share

Reply:
Fine for me. I have my stuff always under /usr/local/... I will correct it.

There is also a bugfix for the firmware, so please wait with merging.